checkov/terraform/graph_builder/local_graph.py

```diff
@@ -286,14 +300,6 @@ def _build_edges_for_vertex(self, origin_node_index: int, vertex: TerraformBlock
             if len(target_variables) == 1:
                 self._create_edge(target_variables[0], origin_node_index, "default", cross_variable_edges)
 
-    def reset_edges(self) -> None:
-        self.edges = []
-        self.out_edges = defaultdict(list)
-        self.in_edges = defaultdict(list)
-        for i in range(len(self.vertices)):
-            self.out_edges[i] = []
-            self.in_edges[i] = []
-
```

Comment:
looks like you deleted it but didn't add this to the new `_arrange_graph_data` method

Reply:
in the `_add_block_data_to_graph` you append to the in_edges and out_edges, but you need to reset it first